### PR TITLE
fix(schedule): error when changing max score

### PR DESCRIPTION
**Pull request by apalchys**
#### 🧑‍⚖️ Pull Request Naming Convention

 - Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
 - Do not put issue id in title
 - Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
 - Consider to add `area:*` label(s)

 * [x] I followed naming convention rules

---

#### 🤔 This is a ...
- [ ] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

https://github.com/rolling-scopes/rsschool-app/issues/816

#### 💡 Background and solution 

TLDR; `courseTaskId` was sent as `taskId` and it triggered the error.
Fixed it and update API

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [ ] Changes are tested locally

### CODE REVIEW

File: server/src/routes/common.ts [hunk 1]
```diff
@@ -34,7 +34,8 @@ export const createPostRoute =
   async (ctx: Router.RouterContext) => {
     const { id, createdDate, ...data } = ctx.request.body;
     try {
-      const result = await getRepository(entity).insert(data);
```
Comment (not-SAINT) [inline server/src/routes/common.ts:37]: I suggest restore this line and send result in response because we need new id to create cousre entity after creating new base task/event.

File: client/src/components/Schedule/FormEntity.tsx [hunk 2]
```diff
@@ -377,20 +377,19 @@ const createTask = async (courseId: number, values: any, isUpdateMode: boolean,
...
+  let taskId = undefined;
 
   if (isUpdateMode && editableRecord) {
-    taskTemplateId = editableRecord.id;
-    await taskService.updateTask(taskTemplateId, templateTaskData);
+    await taskService.updateTask(editableRecord.id, templateTaskData);
   } else {
-    const data: any = await taskService.createTask(templateTaskData);
-    taskTemplateId = data.identifiers[0].id;
+    const { id } = await taskService.createTask(templateTaskData);
```
Comment (not-SAINT) [inline client/src/components/Schedule/FormEntity.tsx:385]: On my local app, I resive InsertResult with identifiers, generatedMaps, raw. And nothing works with  { id } = await changes... I suggest restore previous variant.

File: client/src/components/Schedule/FormEntity.tsx [hunk 3]
```diff
@@ -433,7 +432,7 @@ const createEvent = async (
     await eventService.updateEvent(eventTemplateId, templateEventData);
   } else {
     const data: any = await eventService.createEvent(templateEventData);
-    eventTemplateId = data.identifiers[0].id;
+    eventTemplateId = data.id;
```
Comment (not-SAINT) [inline client/src/components/Schedule/FormEntity.tsx:435]: Same state with event... I suggest restore previous variant.

**Review by not-SAINT — APPROVED** [review]
